src/DataTable/DataTable.svelte

```diff
@@ -132,7 +132,7 @@
       .split(/[\.\[\]\'\"]/)
       .filter((p) => p)
       .reduce(
-        (o, p) => (o && typeof o === "object" && o[p] ? o[p] : defaultValue),
+        (o, p) => ((o && typeof o === "object" && typeof o[p] === 'undefined') ? o[p] : defaultValue),
```

Comment:
Allow whatever value `o` is to render as is if it's not an object.

```suggestion
        (o, p) => ((o && typeof o === "object") ? o[p] : o),
```

Reply:
Code updated. I just found `cell.display` useful also.
Thank @metonym for the change.